cinema/views.py

```diff
@@ -21,7 +23,7 @@
     MovieSessionDetailSerializer,
     MovieListSerializer,
     OrderSerializer,
-    OrderListSerializer,
+    OrderListSerializer, MovieImageSerializer,
```

Comment:
in this case we should not write imports in one line

Reply:
I agree, make this import from new line